insights/parsers/tests/test_candlepin_error_log.py

```diff
@@ -0,0 +1,24 @@
+from insights.tests import context_wrap
```

Comment:
Normally this test would be added to the module test file `parsers/tests/test_foreman_log.py`, is there a reason you created a separate test file?

Reply:
@bfahr  Write the test case under ```parsers/tests/test_foreman_log.py``` file  and delete extra ```parsers/tests/test_candlepin_error_log.py``` file. 
Kindly review...!